### PR TITLE
Add ndm

### PR DESCRIPTION
**Pull request by 45kb**
**By submitting this pull request, I promise I have read the [contributing guidelines](https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

Hi, i added [ndm](https://github.com/720kb/ndm) (Npm Desktop Manager)
It is built with Electron you can be sure about it looking at [this i guess](https://github.com/720kb/ndm/blob/master/index.js#L4)

We released the first demo exactly 20 days ago (GMT+1 :D ) , thought it was great to share it.
The idea is to create a useful tool obviously, that's the main reason why i spread the word. 
It is a Beta release atm, quite stable ( i'd say...)

Thanks for this list which is superuseful and well directed! 
BTW: also many thanks for `require('shell-path').sync()` 🎉

### CODE REVIEW

**Comment by sindresorhus** [comment]
Glad to see someone finally do this. I've been asking people to create a GUI for npm for years.

Some feedback:

- The GPL3 license is unfortunate and will hurt potential contributions.
- The menubar menus are not like native macOS apps. Please read https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/MenuBarMenus.html and see https://github.com/sindresorhus/caprine/blob/43ff3bd34132af89a8970c4d375eda1b8468caa9/menu.js#L100 for how to do it correctly.
- I clicked Globals, and it never finished.
- The bottom statusbar is not useful and only distracting. Those items are better but in the menubar.
- Clicking "Install Package" should focus the package name textfield.
- In the package list table I should be able to use the arrow keys to go to the next and previous package.
- It's `npm`, not `Npm`.

File: readme.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ Made with Electron.
 - [Dext](https://github.com/vutran/dext) - Launcher.
 - [PB for Desktop](https://github.com/sidneys/pb-for-desktop) - Pushbullet client.
 - [GroupMe](https://github.com/dcrousso/GroupMe) - Unofficial GroupMe app.
+- [ndm](https://github.com/720kb/ndm) - Npm desktop manager
```
Comment (sindresorhus) [inline readme.md]: Per guidelines, it should end in a dot.

Npm => npm

Comment (45kb) [inline readme.md]: @sindresorhus yeah sorry, I thought I had read that inside the guidelines, like the description should start with a capital letter, my bad (makes enough sense also to me to read Npm )

Gonna rebase and push the changes, sorry for wasting time on it.

**Comment by 45kb** [comment]
@sindresorhus thanks a lot, yes a GUI was missing probably.
Actually having used Cakebrew for some time ... we noticed how often is more convenient/comfortable.

> The GPL3 license is unfortunate and will hurt potential contributions.

Would you suggest the MIT? May i ask you why if so?
I'd like to better understand this if possible.

> The menubar menus are not like native macOS apps. Please read https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/MenuBarMenus.html and see https://github.com/sindresorhus/caprine/blob/43ff3bd34132af89a8970c4d375eda1b8468caa9/menu.js#L100 for how to do it correctly.

Yes, true! Between the next releases there will be improvements in that sense too, this first release is more to test out the npm.js API that we couldn't find documented anywhere unfortunately....we only found the CLI documentation actually ..... 🎱  Do you even know anything about it ?

> I clicked Globals, and it never finished.

If possible in some way, i'd like to inspect what's happening .... the Globals part as you image, is really a nightmare 🙈 

> The bottom statusbar is not useful and only distracting. Those items are better but in the menubar.

The "footer" ?

> Clicking "Install Package" should focus the package name textfield.
 Ouch .. true... forgot a reminder for that,  i'll open an issue for that 😱 

> In the package list table I should be able to use the arrow keys to go to the next and previous package.
 Y, going to open an issue for that too 👍 

Just saying but if you want to come around and do whatever, seriously anytime!

File: readme.md (hunk 1 above)
Comment (45kb) [inline readme.md]: should be ok now @sindresorhus i hope 👍

**Comment by sindresorhus** [comment]
> Would you suggest the MIT? May i ask you why if so?
I'd like to better understand this if possible.

GPL is a lot more restrictive, which would prevent people from easily including some of your code in their products. This is IMHO against the open source I believe in; Sharing code freely without restrictions. If you want the most people to benefit from your code, MIT or BSD license is the best choice. I would never contribute to a GPL project.

> The first release is more to test out the npm.js API that we couldn't find documented anywhere unfortunately....we only found the CLI documentation ... 🎱 ... Do you know anything about if exists?

There is none. I would strongly recommend using Yarn, for so many reasons.

> The "footer" ? I think i don't understand the element you are referring to... mmm

The one with the links to `Contribute`, `Support`, `Follow`, etc.

**Comment by 45kb** [comment]
@sindresorhus 

> GPL is a lot more restrictive, which would prevent people from easily including some of your code in their products. This is IMHO against the open source I believe in; Sharing code freely without restrictions. If you want the most people to benefit from your code, MIT or BSD license is the best choice. I would never contribute to a GPL project.

I have to admit that, personally, i don't know much about how licenses truly works, we have discussed about it  just a few with @wouldgo but we were quite unsure which to choose.
I really think we will review the type, yes.

> There is none. I would strongly recommend using Yarn, for so many reasons.

Agree, Yarn is damn great, we think to introduce it as a "booster" feature later BUT i have to admit that we would take some time before to introduce it. 
Because really don't have a clear idea yet of what will happen in the future of npm and Yarn unfortunately.

> The one with the links to Contribute, Support, Follow, etc.

Found thanks 🎯 

Just, not to waste your time but: what are your thought about the future of npm and Yarn? Do you think they will coexist or one will prevail over the other (like Yarn seems to do right now) ?

Thanks for merging and for your explanations!

**Comment by sindresorhus** [comment]
> Just, not to waste your time but: what are your thought about the future of npm and Yarn? Do you think they will coexist or one will prevail over the other (like Yarn seems to do right now) ?

npm will never disappear, but Yarn is clearly better. Both use the npm registry.

**Comment by 45kb** [comment]
> npm will never disappear, but Yarn is clearly better. Both use the npm registry.

Truly npm lover ehehe
Why Yarn still not have a own registry it's mistery for me, i see so many great devs involved in the project....

**Comment by 45kb** [comment]
@sindresorhus however, if you accidentally :smile: get through the app code ping me, i am 292929/h on 24h around. appreciated a lot.

**Pull request merged**